client/src/templates/Challenges/components/challenge-title.css

```diff
@@ -48,6 +47,24 @@
   flex-shrink: 0;
 }
 
+.breadcrumb-center {
+  text-decoration: none;
+  min-width: 25px;
+  display: inline-block;
+  align-items: center;
+  justify-content: center;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  max-height: 25px;
+  flex-grow: 1;
+  flex-shrink: 2;
+  background-color: var(--quaternary-background);
+  overflow: hidden;
+  text-overflow: ellipsis;
```

Comment:
`overflow: hidden` and `text-overflow ellipsis` is in the class twice.

Reply:
You know, I think I have deleted that duplicate 3 times now...

Reply:
I will keep an eye out for it :rofl: